Reply/app/src/main/java/com/materialstudies/reply/ui/home/HomeFragment.kt

```diff
@@ -119,10 +132,12 @@ class HomeFragment : Fragment(), EmailAdapter.EmailAdapterListener {
         reenterTransition = MaterialElevationScale(true).apply {
             duration = resources.getInteger(R.integer.reply_motion_duration_large).toLong()
         }
-        val emailCardDetailTransitionName = getString(R.string.email_card_detail_transition_name)
-        val extras = FragmentNavigatorExtras(cardView to emailCardDetailTransitionName)
-        val directions = HomeFragmentDirections.actionHomeFragmentToEmailFragment(email.id)
-        findNavController().navigate(directions, extras)
+        EmailUtils.updateEmailId(email.id)
+
+        // If we are in Small or Medium then we navigate to the email.
+        if (screenSize == SMALL || screenSize == MEDIUM) {
+            goToEmailFragment(cardView, email)
+        }
```

Comment:
Delete and create a public method in MainActivity that opens the pane and gets called here:

`(activity as MainActivity).openEmailDetailsPane()`

On a phone layout, this will slide the email pane in. When showing side-by-side this will be a no-op